src/ebay-page-notice/README.md

```diff
@@ -43,6 +44,8 @@ import "@ebay/skin/page-notice";
 
 | Name         | Type                                                                          | Stateful | Description                                                                                                                                 | Default       |
 | ------------ | ----------------------------------------------------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                     | `"general"` |
+| `status`     | String ( `"general"`,`"attention"`, `"confirmation"`, or `"information"`) | No       | Determines the style and type of notice to be displayed                                                                                         | `"general"`   |
 | `aria-label` | String                                                                        | No       | The description of the notice itself for screen readers. Check out [this issue](https://github.com/eBay/skin/issues/1001) for more context. | -             |
 | `children`   | React Node                                                                    | No       | The content to be displayed within the notice                                                                                               | -             |
+| `a11yDismissText` | String                                                                   | No       | Determines if the notice will have a dismiss button.  Acts as the aria-label for the dismiss button. Should not be used with a footer.      | -             |
+| `onDismissed` | Dispatch<boolean>                                                            | No       | Handler for any extra functionality for after a notice is dismissed.                                                                        | -             |
```

Comment:
I think we've renamed it to `onDismiss`?

Reply:
You're right. Forgot to update this when I made the other change. Updated.